src/threefold_token/token_overview/special_wallets/GA7OPN4A3JNHLPHPEWM4PJDOYYDYNZOM7ES6YL3O7NC3PRY3V3UX6ANM.md

```diff
@@ -0,0 +1 @@
+3340735.94
```

Comment:
Since these files are regenerated on every build and serve, there's no need to commit them (committed versions never get used). Let's gitignore these instead.